pipelines/dax/docker/build_imgserv.groovy

```diff
@@ -13,28 +13,55 @@ node('jenkins-master') {
 }
 
 notify.wrap {
+  def retries = 3
+  def repo = 'webserv/imgserv'
+  def tag = 'dax_latest'
+
+  def run = {
+    dir('imgserv/lsst-dm-ci') {
+        stage('build') {
+            retry(retries) {
+                build(repo)
+            }
+        } // build
+
+        stage('test') {
+            retry(retries) {
+                docker.image(repo:tag).inside {
+                    util.bash "./run_tests.sh"
+                }
+            }
+        } // test
+
+        stage('publish') {
+            util.bash "./pub_image.sh"
+        } // publish
+    } // dir
+  } // run
+
   node('docker') {
-    timeout(time: 4, unit: 'HOURS') {
+    timeout(time: 2, unit: 'HOURS') {
       dir('imgserv') {
         git([
           url: util.githubSlugToUrl('lsst/dax_imgserv'),
           branch: 'master'
         ])
       }
-
-      build('prod_image.sh')
+      try {
```

Comment:
If try statement is to be removed, which I don't mind, how should I go about setting currentBuild.result?

Reply:
It doesn't need to be set at all unless the failure exception is being caught and not rethrown.

Reply:
Try block removed.

Reply:
OK